dockerfiles/init/modules/openshift/files/scripts/che-openshift.yml

```diff
@@ -64,6 +68,8 @@ items:
             name: http
           - containerPort: 8000
             name: http-debug
+          - containerPort: 8888
```

Comment:
Is it default value? Is it a different property from [TCP_PORT](https://github.com/eclipse/che/pull/8982/files#diff-be9e46711b86b2abcc7f8a90858249b3R18)?

Reply:
Yes, thats different one needed  for KUBE_PING.

Reply:
Ok, thanks

Reply:
@mshaposhnik can you elaborate on how it is used? It is not clear for me.

Reply:
Pls read https://github.com/jgroups-extras/jgroups-kubernetes/tree/0.9.3

Reply:
@mshaposhnik  If there is no service for this port how this port can be accessed from other pods?

Reply:
Have you read the doc? there IS a service...

Reply:
Yes, I read the docs. But I mean kubernetes service. Without it there is no access to a port of a pod.

Reply:
I think there is access if you know IP for a container. I think Kubernetes JGroups discovers pods/containers IPs itself.
If you access by service, actually you don't know which pod are you request if replication is more than 1. So, it's impossible to use service at all.
Maybe I'm wrong somewhere, but I imagine it in this way.

Reply:
Yeah, agree about service. 

But how the lib accesses k8s/OS API? which token or SA is used?

Reply:
You can set it into ENV or default will be used from here:
https://github.com/jgroups-extras/jgroups-kubernetes/blob/0.9.3/kube/src/main/java/org/jgroups/ping/kube/KubePing.java#L88

Reply:
Then we need to be sure that this approach works in both kubernetes and openshift before the merge.